GameServer/commands/gmcommands/announce.cs

```diff
@@ -65,16 +93,18 @@ public void OnCommand(GameClient client, string[] args)
 
 						foreach (GameClient clients in WorldMgr.GetAllPlayingClients())
                             if(clients != null)
-							    clients.Player.Out.SendCustomTextWindow(LanguageMgr.GetTranslation(clients, "GMCommands.Announce.WindowAnnounce", client.Player.Name), messages);
+	                            // Message: Announce from {0}
+	                            ChatUtil.SendWindowMessage((int)eWindow.Text, clients, "GMCommands.Announce.WindowAnnounce", messages);

```

Comment:
System.InvalidCastException: Unable to cast object of type 'System.Collections.Generic.List`1[System.String]' to type 'System.String'.
   at DOL.GS.ChatUtil.SendWindowMessage(Int32 type, GameClient target, String title, Object[] args)
   at DOL.GS.Commands.AnnounceCommandHandler.OnCommand(GameClient client, String[] args)
   at DOL.GS.ScriptMgr.ExecuteCommand(GameClient client, GameCommand myCommand, String[] pars)
   at DOL.GS.ScriptMgr.HandleCommand(GameClient client, String cmdLine)

Reply:
After [c09fa47](https://github.com/OpenDAoC/OpenDAoC-Core/pull/30/commits/c09fa4721843aecadac25e9c3f153a651958b5ba):
`InvalidCastException` line 97.